.chloggen/2833-fix-detector-resourcedetectionprocessor.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: collector
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Use the k8snode detector instead of kubernetes for the automatic RBAC creation for the resourcedetector"
```

Comment:
Why the kubernetes does not work? 

is this a breaking change?

Reply:
whas `kubernetes` deprecated or removed from the processor? It would be good to note that

Reply:
No. It was a mistake made by the operator. `kubernetes` has never exist` It was `k8snode`.